tests/util/src/opentelemetry/test/test_base.py

```diff
@@ -44,11 +46,11 @@ def tearDownClass(cls):
     def setUp(self):
         self.memory_exporter.clear()
 
-    def check_span_instrumentation_info(self, span, module):
+    def assertSpanHasInstrumentationInfo(self, span, module):
```

Comment:
Hm, not trying to be nitpicky but this name is misleading. It is not checking that the span has instrumentation info but that the span instrumentation info is the same as the one in the module.

Reply:
Maybe `assertEqualSpanModuleInstrumentationInfo`? :shrug: